project/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,6 +14,10 @@
 
 package com.google.sps.servlets;
 
+import com.google.sps.data.Location;
+import com.google.sps.data.Election;
```

Comment:
nit: Order the imports in alphabetic order. `Election` comes before `Location`.

Reply:
Oops! I tried to order them but messed this one up.